common/service/config/persistence.go

```diff
@@ -66,3 +76,95 @@ func (c *Persistence) Validate() error {
 func (c *Persistence) IsAdvancedVisibilityConfigExist() bool {
 	return len(c.AdvancedVisibilityStore) != 0
 }
+
+// GetConsistency returns the gosql.Consistency setting from the configuration for the store
+func (c *CassandraConsistencySettings) GetConsistency() gocql.Consistency {
```

Comment:
Could also consider moving these to a separate cassandra specific file. Am on the fence.